jwql/jwql_monitors/generate_preview_images.py

```diff
@@ -26,7 +26,7 @@
         python generate_preview_images.py
 """
 
-from glob import glob
+import glob
```

Comment:
@bhilbert4 I changed this to be a bit more consistent with the use of `glob` in our other modules